addons/l10n_ar/models/account_move.py

```diff
@@ -177,17 +177,22 @@ def _reverse_moves(self, default_values_list=None, cancel=False):
     def _inverse_l10n_latam_document_number(self):
         super()._inverse_l10n_latam_document_number()
 
-        # Avoid that user change the POS number (x.l10n_latam_document_number), Rhe POS number configure in journal it
-        # will always be used
         to_review = self.filtered(
             lambda x: x.journal_id.type == 'sale' and x.l10n_latam_document_type_id and x.l10n_latam_document_number and
             (x.l10n_latam_manual_document_number or not x.highest_name))
         for rec in to_review:
             number = rec.l10n_latam_document_type_id._format_document_number(rec.l10n_latam_document_number)
             current_pos = int(number.split("-")[0])
             if current_pos != rec.journal_id.l10n_ar_afip_pos_number:
-                raise UserError(_('Can not change the POS number, you can only change the first number for document'
-                                  ' type that you are creating in odoo'))
+                invoices = self.search([('journal_id', '=', rec.journal_id.id), ('posted_before', '=', True)], limit=1)
+                # If there is no posted before invoices the user can change the POS number (x.l10n_latam_document_number)
+                if (not invoices):
+                    rec.journal_id.l10n_ar_afip_pos_number = current_pos
+                    rec.journal_id._onchange_set_short_name()
+                # If not, avoid that the user change the POS number
+                else:
+                    raise UserError(_('The document number can not be changed for this journal, you can only modify'
```

Comment:
you can only modify the pos number if there are no posted (or posted before) invoiced  Probably still needs translations also... (but let's get this merged already)  Thanks for the follow-up @Bruno-Zanotti

Reply:
@jco-odoo I'm sorry, I completely forget about that. I will create a new task in l10n_ar project and make a new pr for that